[CUBRIDMAN-221] [EN] Reorganize docs related to SQL procedural languages (#619)
CUBRID/cubrid-manual: en/sql/schema/stored_routine_stmt.rst

     'sp_int'                        0  'i'                   'INTEGER'             'IN'

DROP PROCEDURE
==============

In CUBRID, A stored proceudre can be deleted using the **DROP PROCEDURE** statement.
Also, you can delete multiple stored procedures at the same time with several *procedure_name*\s separated by a comma (,).

::

    DROP PROCEDURE procedure_name [{ , procedure_name , ... }];

*   *procedure_name*: Specifies the name of procedure to delete

.. code-block:: sql

    DROP PROCEDURE hello, sp_int;

A stored procedure can be deleted only by the user who published it or by DBA members. 
For example, if a **PUBLIC** user published the 'sp_int' stored procedure, only the **PUBLIC** or **DBA** members can delete it.

DROP FUNCTION
==============

In CUBRID, A stored function can be deleted using the **DROP FUNCTION** statement.

> Also, you can delete multiple stored functions at the same time with several *function_name*\s separated by a comma (,).

Reviewer: ```suggestion
You can also delete multiple stored functions at the same time by specifying several *function_name*\s separated by commas (,).
```
"Also,"의 사용은 틀린건 아니지만 살짝 어색하여 위와 비슷한 수정을 제안합니다.
